docs/source/examples/Catch that asteroid!.ipynb

```diff
@@ -0,0 +1,804 @@
+    }
+   ],
+   "source": [
+    "florence_hte = HeliocentricTrueEcliptic(\n",
+    "    x=florence.r[0], y=florence.r[1], z=florence.r[2],\n",
+    "    d_x=florence.v[0], d_y=florence.v[1], d_z=florence.v[2],\n",
+    "    representation=CartesianRepresentation,\n",
+    "    differential_cls=CartesianDifferential,\n",
+    "    obstime=EPOCH\n",
+    ")\n",
+    "florence_hte"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<ICRS Coordinate: (x, y, z) in km\n",
+       "    (  1.46271267e+08, -53880021.10997667, -20906907.63736058)\n",
+       " (v_x, v_y, v_z) in km / s\n",
+       "    ( 7.40819561,  23.45167568,  24.11931483)>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "florence_icrs_trans = florence_hte.transform_to(ICRS)\n",
+    "florence_icrs_trans.representation = CartesianRepresentation\n",
+    "florence_icrs_trans"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1 x 3 AU x 44.6 deg orbit around Sun (☉)"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "florence_icrs = Orbit.from_vectors(\n",
+    "    Sun,\n",
+    "    r=[florence_icrs_trans.x, florence_icrs_trans.y, florence_icrs_trans.z] * u.km,\n",
+    "    v=[florence_icrs_trans.v_x, florence_icrs_trans.v_y, florence_icrs_trans.v_z] * (u.km / u.s),\n",
+    "    epoch=florence.epoch\n",
+    ")\n",
+    "florence_icrs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(<Quantity [  1.46271267e+08, -5.38800211e+07, -2.09069076e+07] km>,\n",
+       " <Quantity [  7.40819561, 23.45167568, 24.11931483] km / s>)"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "florence_icrs.rv()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us compute the distance between Florence and the Earth:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from poliastro.util import norm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/latex": [
+       "$7066700.8 \\; \\mathrm{km}$"
+      ],
+      "text/plain": [
+       "<Quantity 7066700.8455871325 km>"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "norm(florence_icrs.r - earth.r)"
```

Comment:
ESA has given Florence distance from Earth surface, maybe we could substract Earth radius

Reply:
Good idea! Perhaps that difference of ~6700 km will take us even closer to the announced value :)